### PR TITLE
fix(verifyconditions): also strip http(s) protocol from registry host for split('/')[0]

### PR DESCRIPTION
**Pull request by sjoukedv**
http:// and https:// are also valid host prefixes

Signed-off-by: Sjouke de Vries <info@sdvservices.nl>

### CODE REVIEW

**Comment by nflaig** [comment]
> http:// and https:// are also valid host prefixes

This does not make sense to me, host is not supposed to be a URL in the first place. As far as I know ChartMuseum registries use HTTP protocol but those do not require a login.

Removing the oci:// was just a workaround because helm required it when publishing, so it was added on publish and removed on login.
- see https://github.com/nflaig/semantic-release-helm/pull/14

Why is this not solved by `skipRegistryLogin`?

File: lib/verifyConditions.js [hunk 1]
```diff
@@ -14,7 +14,11 @@ module.exports = async (pluginConfig, context) => {
     const registryHost = env.REGISTRY_HOST || pluginConfig.registry;
 
     if (registryHost && !pluginConfig.skipRegistryLogin && !pluginConfig.isChartMuseum && env.REGISTRY_USERNAME && env.REGISTRY_PASSWORD) {
-        const registryUrl = registryHost.replace("oci://", "").split('/')[0];
+        const registryUrl = registryHost
```
Comment (nflaig) [inline lib/verifyConditions.js:17]: The variable name is confusing, let's rename it

```suggestion
        const domain = registryHost
```

**Comment by sjoukedv** [comment]
Maybe my use case is too specific I have a mono repo with a container image and helm chart that are pushed to the same registry. Docker registries do support http(s)://, I haven't checked helm docs but just assumed we are striping the protocol from string because of the `split('/')[0]` (basically grabbing the host without the path) which does not work if you don't remove http(s):// first

**Comment by nflaig** [comment]
> Maybe my use case is too specific

It fine, we can support this case as well. Maybe others find it useful in the future. Just trying to understand the reasoning behind changes.

**Review by nflaig — APPROVED** [review]

**Pull request merged**